classes/core/PKPContainer.inc.php

```diff
@@ -167,8 +171,17 @@ protected function loadConfiguration()
             'retry_after' => 90,
         ];
 
+        // Logging
+        $items['logging']['channels']['mail'] = [
+            'driver' => 'single',
+            'path' => Config::getVar('email', 'debug_path') ??
+                (new PrivateFileManager())->getBasePath() . '/' . 'email.log',
+            'level' => 'debug',
+        ];
+
         // Mail Service
-        $items['mail']['default'] = Config::getVar('email', 'smtp') ? 'smtp' : 'sendmail';
+        $items['mail']['default'] = Config::getVar('email', 'debug') ? 'log' :
```

Comment:
This could be a function with early return logic, instead of a nested ternaries

Reply:
@henriqueramos can you provide an example?

Reply:
Sure! Could be something like this.

```
protected function getDefaultMailer (): string {
    if (Config::getVar('email', 'debug')) {
        return 'log';
    }

    if (Config::getVar('email', 'smtp')) {
        return 'smtp';
    }

    return 'sendmail';
}
```